modules/developers-guide/pages/tutorials/counter-tutorial.adoc

```diff
@@ -207,29 +206,30 @@ dfx canister call my_counter set '(987)'
 dfx canister call my_counter get
 ----
 +
-Returns the current value of 987.
+This returns the updated value of the `+currentValue+` to be 987.
+Running the additional commands
 +
 [source,bash]
 ----
 dfx canister call my_counter increment
 dfx canister call my_counter get
 ----
 +
-Returns the incremented value of 988.
+returns the incremented `+currentValue+` of 988.
 
 include::example$candid-ui.adoc[]
 +
 image:candid-counter.png[Counter functions]
 
-== Stop the local network
+== Stop the local {IC} blockchain
 
-After you finish experimenting with your program, you can stop the local Internet Computer network so that it doesn’t continue running in the background.
+After you finish experimenting with your dapp, you can stop the local {IC} blockchain so that it doesn’t continue running in the background.
```

Comment:
```suggestion
After you finish experimenting with your dapp, you can stop the local canister execution environment so that it doesn’t continue running in the background.
```